PokeAlarm/Manager.py

```diff
@@ -999,6 +1006,15 @@ def add_optional_travel_arguments(self, info):
         if self.__api_req['DRIVE_DIST']:
             info.update(**self.get_driving_data(lat, lng))
 
+    # Add gym details to an info object
+    def add_gym_details(self, info, gym_id):
```

Comment:
yeah I'm not familiar enough with Python to think of those cases. 

But why can't that method still stay to minimize code copy paste? Just with your suggested changes